trymito.io/pages/spreadsheet-automation.tsx

```diff
@@ -0,0 +1,334 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import Image from 'next/image';
+import Footer from '../components/Footer/Footer';
+import Header from '../components/Header/Header';
+import pageStyles from '../styles/Page.module.css';
+import textImageSplitStyles from '../styles/TextImageSplit.module.css';
+import titleStyles from '../styles/Title.module.css';
+import trifoldStyles from '../styles/Trifold.module.css';
+import functionalityCardStyles from '../styles/FunctionalityCard.module.css';
+import spreadsheetAutomationStyles from '../styles/SpreadsheetAutomation.module.css';
+
+import DownloadCTACard from '../components/CTACards/DownloadCTACard';
+import { classNames } from '../utils/classNames';
+import CTAButtons from '../components/CTAButtons/CTAButtons';
+import PivotTableIcon from '../public/step-icons/PivotTableIcon.svg';
+import GraphIcon from '../public/step-icons/GraphIcon.svg';
+import FunctionIcon from '../public/step-icons/FunctionIcon.svg';
+import SettingsIcon from '../public/step-icons/SettingsIcon.svg';
+import ImportIcon from '../public/step-icons/ImportIcon.svg';
+import FilterIcon from '../public/step-icons/FilterIcon.svg';
+
+import Link from 'next/link';
+import { useState } from 'react';
+
+
+
+const SpreadhseetAutomation: NextPage = () => {
+
+  const [selectedFunctionalityCard, setSelectedFunctinoalityCard] = useState<number>(0);
+
+  return (
+    <>
+      <Head>
+        <title>No-Code Spreadsheet Automation Software for Python | Mito</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="Mito is a notebook-native Python library that enables simple, no-code spreadsheet automation with Pandas code generation and AI tools built in. | Mito" />
+      </Head>
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+          <section className={classNames(titleStyles.title_card, pageStyles.background_card, spreadsheetAutomationStyles.flex_col_mobile_row_desktop)}>
+            <div className={spreadsheetAutomationStyles.title_text_container}>
+              <h1>
+                The No-Code Approach to Spreadsheet Automation 
+              </h1>
+              <p>
+                Automate your spreadsheets in hours, not months.
```

Comment:
I could see making this the top line, and the other one is below.

Reply:
I agree. But unfortunetly this is where the SEO constraints come in. For now, at least, let's stick with the ones that they provided to us. If we feel strongly about any of them we can ask them to give us new titles for specific ones.